scripts/check_compliance.py

```diff
@@ -614,6 +617,7 @@ def report_to_github(repo, pull_request, sha, suite, docs):
 
     return comment_count
 
+CURRENT_COMMIT_RANGE = "HEAD~1.."
```

Comment:
`TIP_COMMIT_ONLY(_RANGE)` or the like might be clearer. I misread `CURRENT_COMMIT_RANGE` as "currently active commit range" and was confused.

Reply:
I did struggle to find a good name because it's a "single commit range" is a bit hard to express tersely. How about "HEAD_ONLY_RANGE"?

Reply:
Fine with me.

My check is "can this be read in more than one way?" `CURRENT_COMMIT_RANGE` can be read as `CURRENT_(COMMIT_RANGE)` and `(CURRENT_COMMIT)_RANGE`. `HEAD_ONLY_RANGE` is more difficult to read in two ways.